### PR TITLE
reset type to taylorgram

### PR DESCRIPTION
**Pull request by doutriaux1**

### CODE REVIEW

**Comment by doutriaux1** [comment]
@mcenerney1 that is the bit you need

File: vistrails/gui/uvcdat/diagnosticsDockWidget.py [hunk 1]
```diff
@@ -852,7 +852,10 @@ def displayCell(self, res30, row, column, sheet="Sheet 1"):
          from packages.uvcdat_cdms.init import get_canvas, get_gm_attributes, original_gm_attributes
          from gui.uvcdat.uvcdatCommons import gmInfos
          Gtype = res30.type
-         G = V[Gtype]
+         if res30.type=="Taylor":
+             G=V["Taylordiagram"]
```
Comment (remram44) [inline vistrails/gui/uvcdat/diagnosticsDockWidget.py:856]: Is `V` a `variable`? `G` something graphics related? `pvars` short for `plot_variables`?

Comment (doutriaux1) [inline vistrails/gui/uvcdat/diagnosticsDockWidget.py:856]: @painter1 would know i think
but the code says:

```
         pm = projectController.plot_manager
         V=pm._plot_list["VCS"]
```

so V seems to be the list of plot type in vcs (???)
and 

```
             G=V["Taylordiagram"]
```

seems to indicate the the vcs graphic method type...

**Comment by aashish24** [comment]
I would suggest that we don't use single char variable names. Searching and reading code becomes difficult when we use single char variable names. Can we change it please?

**Comment by doutriaux1** [comment]
Sure but that's not my code, I'm not touching this. I just made it work.

**Pull request merged**